common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,318 @@
+"""
+Acceptance Tests for Course Information
+"""
+import uuid
+
+from bok_choy.web_app_test import WebAppTest
+from common.test.acceptance.pages.studio.course_info import CourseUpdatesPage
+from flaky import flaky
+
+from ...pages.studio.auto_auth import AutoAuthPage
+from ...pages.studio.index import DashboardPage
+from ...pages.studio.overview import CourseOutlinePage
+
+
+def _create_course(self):
+    """
+    Helper method to create a course within the setup method
+    """
+    self.auth_page.visit()
+    self.dashboard_page.visit()
+    self.dashboard_page.wait_for_page()
+    self.assertFalse(self.dashboard_page.has_course(
+        org=self.course_org,
+        number=self.course_number,
+        run=self.course_run
+    ))
+    self.assertTrue(self.dashboard_page.new_course_button.present)
```

Comment:
I would not expect you to test course creation here. That should happen in its own test (and hopefully does!). Extend from StudioCourseTest, which will create a course for you.